src/actions/utils/breakpoints.js

```diff
@@ -44,10 +24,3 @@ export async function getGeneratedLocation(source, sourceMaps, location) {
 
   return await sourceMaps.getGeneratedLocation(location, source.toJS());
 }
-
-export function equalizeLocationColumn(location, referenceLocation) {
-  if (referenceLocation.column) {
-    return location;
-  }
-  return { ...location, column: undefined };
-}
```

Comment:
i think this method is the source of a lot of client miscommunications because the bpClient cannot be found if the location has changed.